core/domain/user_services.py

```diff
@@ -396,7 +396,8 @@ def fetch_gravatar(email: str) -> str:
     else:
         if response.ok:
             if imghdr.what(None, h=response.content) == 'png':
-                return utils.convert_png_binary_to_data_url(response.content)
+                return utils.convert_png_or_webp_binary_to_data_url(
+                    response.content)
```

Comment:
Unneeded change.

Reply:
The line becomes more than 80, that is why I did that.